packages/fluent/scss/core/_variables.scss

```diff
@@ -12,6 +12,50 @@ $use-input-spinner-width: true !default;
 $use-input-spinner-icon-offset: false !default;
 $use-picker-select-width: true !default;
 
+// Spacing
+$kendo-spacer: 1em !default;
+$kendo-spacer-x: $kendo-spacer !default;
+$kendo-spacer-y: $kendo-spacer !default;
+
+$kendo-spacing: (
```

Comment:
We have some paddings (buttons, inputs, etc) that are 6px. Maybe we should add a spacing with 2px step instead of 4px